Kiosk/Bid Fulfillment/LoadingViewController.swift

```diff
@@ -142,6 +142,7 @@ extension LoadingViewController {
 
         let title = reserveNotMet ? "NO, THANKS" : (createdNewBidder ? "CONTINUE" : "BACK TO AUCTION")
         backToAuctionButton.setTitle(title, forState: .Normal)
+        fulfillmentContainer()?.cancelButton.hidden = false
```

Comment:
Yes, the whole `fulfillmentContainer()` is a poor abstraction that I've been meaning to replace when I get a chance. This added line is really fulfilling the promise in this comment: https://github.com/artsy/eidolon/blob/183210ea6c470e5819118ac0fda2d61fe5bc8a05/Kiosk/Bid%20Fulfillment/LoadingViewController.swift#L61